core/transaction_pool/clock.hpp

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_CLOCK_HPP
+#define KAGOME_CLOCK_HPP
+
+#include <chrono>
+
+namespace kagome::transaction_pool {
+
+  /**
+   * An interface for a clock
+   */
+  class Clock {
+   public:
+    /**
+     * Difference between two time points
+     */
+    using Duration = std::chrono::milliseconds;
+    /**
+     * A moment in time, stored in milliseconds since Unix epoch start
+     */
+    using TimePoint = std::chrono::milliseconds;
```

Comment:
milliseconds are actually not a time point, but a duration.

Reply:
I know, I use them this way because it allows to generalize the clock. A unified way to store a time point.